cypress/integration/channel/edit_message_spec.js

```diff
@@ -109,4 +109,46 @@ describe('Edit Message', () => {
             });
         });
     });
+    it('M13542 Open edit modal immediately after making a post', () => {
+        // 1. Login as "user-1" and go to /
+        cy.apiLogin('user-1');
+        cy.visit('/');
+
+        // 2. Post message "Hello"
+        cy.get('#post_textbox').type('Hello').type('{enter}');
+
+        // 3. Post a second message "World!"
+        // 4. Hit 'enter' key and then immediately (as FAST as you can) hit 'arrow-up' to open the edit box
+        cy.get('#post_textbox').type('World!{enter}{uparrow}');
+        cy.get('#edit_textbox').
+            invoke('val').
+            then((val) => { // eslint-disable-line
+                const editText = val;
+                if (editText === 'Hello') {
+                    // 5. If pressed while the new post was still pending, observe the edit box opens for the first post
+                    cy.get('#editPostModal').should('be.visible');
+                    cy.get('#edit_textbox').type(' New message{enter}');
+                    cy.get('#editPostModal').should('be.not.visible');
+
+                    // Check the first post and verify that it contains new edited message.
+                    cy.get('.post-message__text').eq(-3).should('contain', 'Hello New message');
+
+                    // Check that the second post remains unchanged with the original message.
+                    cy.get('.post-message__text').last().should('contain', 'World!');
+                } else if (editText === 'World!') {
+                    // 6. If the edit box open up for the second post, Enter edit text and hit enter, observe the second
+                    // message is changed and the first message is unchanged
+                    cy.get('#editPostModal').should('be.visible');
+
+                    cy.get('#edit_textbox').type(' New message{enter}');
+
+                    cy.get('#editPostModal').should('be.not.visible');
+
+                    cy.get('.post-message__text').eq(-3).should('contain', 'Hello');
+
+                    // Check the second post and verify that it contains new edited message.
+                    cy.get('.post-message__text').eq(-2).should('contain', 'World! edited message');
```

Comment:
Should this be ``('contain', 'World! New message')``?

Reply:
oops, my bad
changed now